src/screens/Notifications/components/NotificationCard/NotificationCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {View, Text, Image} from 'react-native';
+
+import generateName from 'sillyname';
+
+import Thermometer from '../../../../assets/images/Thermometer';
+import Drop from '../../../../assets/images/Drop';
+
+import styles from './notificationCardStyles';
+
+interface IProps {
+	notification: {
+		id: number;
+		type: string;
+		date: string;
+		time: string;
+		operation: string;
+		temperature: string;
+		humidity: string;
+		weekly: boolean;
+		description?: string;
+	};
+}
+
+const randomName = generateName();
+const URI = 'http://i.pravatar.cc/200';
+
+const NotificationsCard: React.FC<IProps> = ({notification}) => {
+	const {date, time, operation, temperature, humidity, weekly, description} = notification;
+
+	return(
+		<View style={styles.container}>
+			<View style={styles.dataWrapper}>
+				<Text style={styles.date}>{date}</Text>
+			</View>
+			<View style={styles.contentWrapper}>
+				<View style={styles.userInfo}>
+					<Image style={styles.image} source={{uri: URI}} />
+					<View style={styles.info}>
+						<Text style={styles.time}>{time}</Text>
+						<Text>
+							<Text style={styles.textBold}>Manager: </Text>
+							<Text>{randomName}</Text>
+						</Text>
+						<Text>
+							<Text style={styles.textBold}>Operation: </Text>
+							<Text style={{color: operation === 'Scheduled' ? '#1ca36e' : '#2582e2'}}>{operation}</Text>
+						</Text>
+					</View>
+				</View>
+				<View style={styles.indicatorWrapper}>
+					<View style={styles.thermometer}>
+						<Thermometer fill='#000000' width={28} height={28} />
```

Comment:
move it to colors variables, and check it in all project (variations - #000 #000000)

Reply:
good catch. We will have a separate task for that right away